docs/introduction.md

```diff
@@ -9,15 +9,17 @@ slug: /introduction
 
 ![kwil-banner](../docs/kwil-banner.svg)
 
-## The permissionless SQL database for Web3-enabled applications
+## Web3 Native Decentralized Database Networks
 
-Kwil is the first permissionless database that enables structured data storage across a decentralized network. Kwil extends the functionality of traditional relational databases by providing a layer on-top of existing engines that enforce rules necessary for building secure, permissionless data stores.
+Kwil is a framework for deploying relational databases as standalone, byzantine fault tolerant networks. Kwil extends the functionality of traditional relational databases by providing a practical byanztine fault tolerant (pBFT) layer on top of existing engines, enforcing rules necessary for building secure, decentralized data stores.
+
+Each Kwil network runs its own permissionless Proof-of-Stake (PoS) or permissioned Proof-of-Authority (PoA) blockchain, enabling projects to have a high-degree of configurability over their networks. With Kwil, applications and products can determine who, where, and how the nodes are run in their blockchain.
```

Comment:
`who` here seems grammarly incorrect? I maybe be wrong

Reply:
You are correct.